codice/ddf
# DDF-5255 Add internationalization to instances of  "Form" in Search Forms

#### What does this PR do?
Adds a single point over control over all instances of "Search Form"

#### Who is reviewing it? 
@Bdthomson
@bellcc  

#### Select relevant component teams:
@codice/ui

#### Ask 2 committers to review/merge the PR and tag them here.
@djblue
@rzwiefel

#### How should this be tested?
NOTE: Instead of "Look Shape" in the following images, you should see "Search [your alt word]"

1. pull, replace the values for form.title and forms.title with an alternate (in title case) for "Form", build, run
1. Verify that all locations, in search forms, where "form(s)"  existed are replaced with your alternate. See screenshots below for locations that I'm aware of.
1. Extra credit for removing a letter from instances of form(s).title to force the "Search Forms" alt text.

#### Any background context you want to provide?
We would like the ability to alter this downstream.

#### What are the relevant tickets?
Fixes: #5255 

#### Screenshots
<img src="https://user-images.githubusercontent.com/22667297/63539779-b4acab00-c4cf-11e9-8fca-faa931af3dae.png" height=200 />
<img src="https://user-images.githubusercontent.com/22667297/63539780-b4acab00-c4cf-11e9-9ff3-69d7ac581c2a.png" height=200 />
<img src="https://user-images.githubusercontent.com/22667297/63539781-b5454180-c4cf-11e9-8e7b-ddb0b1ac2380.png" height=200 />
<img src="https://user-images.githubusercontent.com/22667297/63539782-b5454180-c4cf-11e9-97c0-87f0d3f8d2f4.png" height=200 />
<img src="https://user-images.githubusercontent.com/22667297/63539784-b5ddd800-c4cf-11e9-929e-154f8428e907.png" height=200 />
<img src="https://user-images.githubusercontent.com/22667297/63539786-b6766e80-c4cf-11e9-982b-3caa271039a1.png" height=200 />
<img src="https://user-images.githubusercontent.com/22667297/63539789-b6766e80-c4cf-11e9-9b6a-baa48378fffd.png" height=200 />

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
